remind_class.py

```diff
@@ -0,0 +1,77 @@
+import email, smtplib, ssl
```

Comment:
The best practice is to put each import on a separate line.

PEP 8 is a python style guide that has guidelines for best formatting practices. Here are their comments on imports:

https://peps.python.org/pep-0008/#imports